pkg/server/server.go

```diff
@@ -801,7 +801,7 @@ func validateAdmin(imageSpace string, w http.ResponseWriter, r *http.Request) bo
 	if err != nil {
 		message := fmt.Sprintf("Unable to get permission token %s", err)
 		kiln.LogError.Printf(message)
-		writeErrorResponse(http.StatusUnauthorized, message, w)
```

Comment:
It would had been better to conditionally do this.  Not all errors will be `403`.  It might even make sense to just proxy through the request's status through to the user as well.  Thoughts?